shpc/main/modules/module.py

```diff
@@ -40,16 +55,6 @@ def add_environment(self):
             environment_file=self.settings.environment_file,
         )
 
-    def validate_tag_exists(self):
-        """
-        Ensure that a provided module name (and tag) exists.
-        """
-        if not self.config.tag:
```

Comment:
Let's keep this here - to follow up from the above, we want to be able to load a config that has a tag as None without it exiting. However here, we expect it to fail!